zerver/openapi/zulip.yaml

```diff
@@ -6725,6 +6725,13 @@ paths:
                           Present if `realm` is present in `fetch_event_types`.
 
                           Whether messages can be deleted in this Zulip organization.
+                      realm_oss_showcase_policy:
+                        type: integer
+                        description: |
+                          Present if `realm` is present in `fetch_event_types`.
+
+                          The policy for whether the organization is publicly listed on on
+                          zulip.com/communities.
```

Comment:
I rewrote the text here to provide a bit more clarity that most API developers will want to ignore this field.